.circleci/config.yml

```diff
@@ -72,14 +36,80 @@ jobs:
           name: Upload package to Pypi if functional changes were made
           command: ./.circleci/deploy-if-version-bump.sh
 
+  build_python2:
+    docker:
+      - image: python:2.7.14
+
+    steps:
+      - checkout
+
+      - restore_cache:
+          key: v1-py2-{{ checksum "setup.py" }}
+
+      - run:
+          name: Create a virtualenv
+          command: |
+            mkdir -p /tmp/venv/aotearoa
+            virtualenv /tmp/venv/aotearoa
+            echo "source /tmp/venv/aotearoa/bin/activate" >> $BASH_ENV
+
+      - run:
+          name: Install dependencies
+          command: |
+            pip install --upgrade pip twine wheel
+            # pip install --editable git+https://github.com/openfisca/openfisca-core.git@BRANCH#egg=OpenFisca-Core[web-api]  # use a specific branch of OpenFisca-Core
+            pip install --editable .[dev] --upgrade
+
+      - save_cache:
+          key: v1-py2-{{ checksum "setup.py" }}
+          paths:
+            - /tmp/venv/aotearoa
+
+      - run:
+          name: Run tests
+          command: make test
+
+  build_python3:
+    docker:
+      - image: python:3.7
```

Comment:
@Br3nda is this going to build python 3.6 or 3.7? Changelog says 3.6

Reply:
good catch! i ended up copying 3.7 from Uruguay. I'll fix the changelog